filebeat/docs/modules/coredns.asciidoc

```diff
@@ -27,6 +27,18 @@ This module comes with a sample dashboard.
 [role="screenshot"]
 image::./images/kibana-coredns.jpg[]
 
+[float]
+==== `log` fileset settings
+
+Example config:
+
+[source,yaml]
+----
+- module: corends
+  log:
+    enabled: true
```

Comment:
Would an example with `var.paths:` included be better here? Or just a commented out reference for the shake of completeness. wdyt?

Reply:
Thanks you for your suggestion. I've extended the example.